setup-libraries.ipynb

```diff
@@ -31,13 +260,17 @@
     "if [ -d cmssw-env ]; then\n",
     "    rm -rf cmssw-env\n",
     "fi\n",
-    "cd $HOME\n",
-    "scramv1 project -n \"${CMSSW_VER}_Jets1\" $CMSSW_VER \n",
+    "if [ ! -f \"$HOME/$DIRFORCHECKOUT\" ]; then\n",
+    "    echo \"$HOME/$DIRFORCHECKOUT does not exist. Create it\"\n",
+    "    mkdir \"$HOME/$DIRFORCHECKOUT\"\n",
+    "fi\n",
+    "cd \"$HOME/$DIRFORCHECKOUT\"\n",
+    "scramv1 project -n \"${CMSSW_VER}\" $CMSSW_VER \n",
     "\n",
-    "cd \"${CMSSW_VER}_Jets1/src\"\n",
+    "cd \"${CMSSW_VER}/src\"\n",
     "eval `scramv1 runtime -sh`\n",
     "git clone https://github.com/cms-jet/JMEDAS.git Analysis/JMEDAS -b HATS2020\n",
-    "git clone https://github.com/cms-jet/JetToolbox Analysis/JetToolbox -b jetToolbox_102X_v3\n",
+    "git clone https://github.com/cms-jet/JetToolbox Analysis/JetToolbox -b jetToolbox_102X\n",
```

Comment:
Why did you revert the branch used by the Toolbox?

Reply:
jetToolbox_102X_v3 doesn't exist. Not sure why it was changed to [non-existing] _v3 before. With the _v3 it gives a detached head state

Reply:
jetToolbox_102X_v3 is a tag, not a branch. detached head is fine. it doesn't mean the tag doesn't exist.

Reply:
I changed this to follow the directions in the Jet Toolbox README:

```
git clone git@github.com:cms-jet/JetToolbox.git JMEAnalysis/JetToolbox -b jetToolbox_102X_v3
```

It looks like this is a tag, rather than a branch, which causes the detached HEAD state.

Reply:
Also, the tag and the branch are identical right now, so no matter :)

Reply:
Still better to use the tag to make sure it continues to work, if there are changes to the Toolbox 102-branch. Will make a mini-PR to revert. Not sure why I got confused by the detached head this time.